include/cpp-gray/gray.h

```diff
@@ -73,6 +74,16 @@ namespace cppgray
         constexpr explicit gray_code(value_type value) noexcept;
 
         /**
+         * @brief Construction from a `bitset`.
+         *
+         * The exact bit representation of the set is used
+         * as the grey code.
+         *
+         * @param value `bitset` to convert, assumes 8-bit byte.
+         */
+        constexpr explicit gray_code(const std::bitset<sizeof(value_type) * 8> & value) noexcept;
```

Comment:
And here I guess that I'd take a `const std::bitset<N>&` and `static_assert` or SFINAE out those that are greater than `std::numeric_limits<value_type>::digits` but still allow construction from `std::bitset<N>` when `N` is smaller.